misc/package/Data Files/MWSE/core/lib/dependencyManagement/types/mwse.lua

```diff
@@ -0,0 +1,30 @@
+return {
+    id = "mwse",
+    checkDependency = function(self, dependency)
+        if not dependency.buildnumber then return true end
+        self.logger:debug("Checking MWSE version: %s", dependency.buildnumber)
+        self.logger:debug("Current MWSE version: %s", mwse.buildNumber)
+        if dependency.buildnumber > mwse.buildNumber then---@diagnostic disable-line: undefined-field
+            self.logger:error("MWSE is out of date")
+            return false, {
+                {
+                    title = "MWSE Update Required",
+                    reasons = {
+                        string.format("Current Build: %d; Required: %s",
+                            mwse.buildNumber, dependency.buildnumber)---@diagnostic disable-line: undefined-field
+                    },
```

Comment:
I documented `mwse.buildNumber` field in #385. Once we get that in, we can remove these `---@diagnostic disable-line: undefined-field`.